src/status_im/contexts/onboarding/events.cljs

```diff
@@ -67,27 +67,15 @@
  (fn [_ [error]]
    {:dispatch [:biometric/show-message (ex-cause error)]}))
 
-;; Navigate to next screen once onboarding animation and account creation both are complete
 ;; Note: If account creation fails, user will get stuck on loading screen
 ;; TODO: Show error screen in case account creation/restoration fails
 (rf/reg-event-fx :onboarding/account-creation-complete
- (fn [{:keys [db]} [opts]]
-   (let [loading-animation-complete? (or (:loading-animation-complete? opts)
-                                         (:onboarding/loading-animation-complete? db))
-         loading-screen              (or (:loading-screen opts) (:onboarding/loading-screen db))
-         login-signal-received?      (or (:login-signal-received? opts)
-                                         (:onboarding/login-signal-received? db))]
-     (if (and loading-animation-complete? login-signal-received?)
-       {:db (-> db
-                (dissoc :onboarding/loading-screen)
-                (dissoc :onboarding/loading-animation-complete?)
-                (dissoc :onboarding/login-signal-received?))
-        :fx [[:dispatch
-              [:navigate-to-within-stack [:screen/onboarding.enable-notifications loading-screen]]]]}
-       {:db (-> db
-                (assoc :onboarding/loading-screen loading-screen)
-                (assoc :onboarding/loading-animation-complete? loading-animation-complete?)
-                (assoc :onboarding/login-signal-received? login-signal-received?))}))))
```

Comment:
Are we still handling the wait for the `login-signal`? I think we don't wait for the login signal to be received there can be an exception that occurs when pressing a button on the enable-notifications screen

Reply:
hi @seanstrom, Thank you very much for reviewing the PR.

> there can be an exception that occurs when pressing a button on the enable-notifications screen

Actually, that's why we need to wait for the login signal. We need to ensure the account exists before trying to update by enabling notifications. We introduced this fix in https://github.com/status-im/status-mobile/pull/21496

Reply:
Hey @Parveshdhull 👋, yup yup the fix in #21496 is what I'm referring to, is that fix still working with these changes?

Reply:
yes its still working

Reply:
Thank you for bringing attention to this part of the code.

:onboarding/account-creation-complete

It is now just an unnecessary wrapper since we removed the wait for the animation to complete. So, I’ve completely removed it and called this event directly from the login to avoid confusion. Please let me know if it looks good now.

@pavloburykh Please ignore this commit; it’s a minor cleanup change and should not affect testing. (I will self-test the PR before merging.)

Reply:
@Parveshdhull yup yup the code looks good to me ✅